index.html

```diff
@@ -236,29 +236,25 @@ <h3>Stoica Robert-Cristian</h3>
 							<div class="custom-card">
 								<div class="avatar-wrapper">
 									<img
-										src="https://media.istockphoto.com/vectors/thumbnail-image-vector-graphic-vector-id1147544807?k=6&m=1147544807&s=612x612&w=0&h=8CXEtGfDlt7oFx7UyEZClHojvDjZR91U-mAU8UlFF4Y="
+										src="https://i.ibb.co/TLhMXMX/326520486-513017780710409-1049115559909250766-n.jpg"
 										alt=""
 										class="avatar"
 									/>
 								</div>
 								<div>
-									<h3>Full name</h3>
+									<h3>Caramida-Stoian Petrisor Cristian</h3>
 									<p>
-										Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-										Molestiae omnis nobis quo dicta culpa eum exercitationem
-										provident ex iure pariatur? Tempora sunt accusamus tempore
-										modi. Quidem, corrupti perspiciatis aliquid dicta cum odio
-										sapiente tempora quas incidunt adipisci alias praesentium et
-										excepturi error maxime aspernatur sint qui doloremque
-										distinctio. Ex, unde.
+										Sunt un tanar pasionat de it, dornic de a se dezvolta pe plan profesional. Studiez computer-sience in cadroul 
+										facultatii de Automatica si Calculatoare UPB, imi plac provocarile si munca in echipa. Sunt o persoana sociabila,
```

Comment:
Aici ai 2 mici greseli de tipografie, ar trebui 'Studiez computer-science in cadrul'